src/main/java/core/basesyntax/NumberSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class NumberSupplier {
+    public int getRandomNumber() {
+        Random randomNumber = new Random();
```

Comment:
1. confusing name - it's not random number. it's random number GENERATOR
2. make it as `private final` class-level field, no need to create it each time when we call this method 
![image](https://github.com/mogorovskiy/jv-lottery/assets/73041091/5309ce7c-b880-4394-b2d6-e743d0e6f534)

```suggestion
        Random random = new Random();
```